README.md

```diff
@@ -385,6 +355,13 @@ The official Meteor resources page can be found [here](https://www.meteor.com/to
 
 * [We Work Meteor](https://www.weworkmeteor.com/)
 
+## Related
+
+* [Awesome Meteor Developers](https://github.com/harryadelb/awesome-meteor-developers)
```

Comment:
that's awesome ;)
do you think they would want to add those resources here as well?
I would love to add everything in there

Reply:
I appreciate it but I think it's better to keep this repo strictly for Meteor stuff only with links to other stuff.

Reply:
sure, just to say that their https://github.com/harryadelb/awesome-meteor-developers#packages section could probably fit here as well.

Reply:
I see, where we'd display links to the packages and ways of supporting them too. lol, I might have been mistaken. :sweat_smile: